src/data/modifiers.txt

```diff
@@ -5313,6 +5313,7 @@ Effect	Buttery, Boy	Sleaze Resistance: +3, Sleaze Damage: +30, Moxie Percent: +1
 Effect	Buttons Rouged, Louisiana	Avatar: "man with the red buttons"
 Effect	Buy!  Sell!  Buy!  Sell!	Meat Drop: [max(0,202-2*T)]
 Effect	Buzzard Breath	Stench Damage: +30, Spooky Damage: +30
+# Buzzed on Distillate: Variable
```

Comment:
This is variable but you can read it from desc_effect?
Well, since you said you are not parsing it, I guess tgat us out if scope. 😃